plasmapy/atomic/nuclear.py

```diff
@@ -72,6 +72,43 @@ def nuclear_binding_energy(
     return particle.binding_energy.to(u.J)
 
 
+@particle_input(any_of={'isotope', 'baryon'})
+def nuclear_mass_energy(
```

Comment:
Interestingly this is literally what our docs say you should do:

> Follow the existing coding style within a subpackage.

Definitely time to update these docs! :laughing: If you don't mind I'll make a minor stylistic pass over these files before merging.